fix(mobile): removed iOS install content (#13836)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

---

> title:  Install Android apps manually with Gradle and Android Studio

Reviewer: ```suggestion
title: Manually install the Android agent (Gradle/Android Studio)
```
Just an idea!

  - /docs/mobile-monitoring/new-relic-mobile-android/install-configure
---

> To install and configure New Relic functionality with your Android applications, we recommend you follow our [guided install](https://onenr.io/0kjnEoryzRo), located directly in the UI. Using this method, you can install and configure the Android agent with Gradle. these procedures. If you'd prefer to configure your install manually, follow along below:

Reviewer: ```suggestion
Our [guided install](https://onenr.io/0kjnEoryzRo) is the preferred method of installation, but if you need to manually install and configure our Android agent, follow the instructions below.
```

As part of the installation process for mobile monitoring, New Relic automatically generates an [application token](/docs/mobile-apps/viewing-your-application-token). This is a 40-character hexadecimal string for authenticating each mobile app you monitor in New Relic.

1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile**.
2. If applicable: From the [**Mobile apps** index](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-index), select <Icon name="fe-plus-circle"/>
3. From the **Get started** page, select **Android** as the platform for mobile monitoring.
4. Enter a name for your mobile app, and select **Continue**.

> Follow the steps below to configure the Android agent with Gradle and Android Studio.

Author: ```suggestion
After installing the New Relic Android agent, you will need to make some configuration changes to your Android app in order to successfully report data to New Relic. Follow the steps below in your app's Android Studio project:
```

> Continue with the steps to [configure mobile monitoring](#configuration) with Gradle and Android Studio

Reviewer: ```suggestion
Follow the steps below to configure the Android agent with Gradle and Android Studio.
```

## Configure with Gradle and Android Studio [#configuration]

> These procedures to configure your Android app with Gradle and Android Studio also appear on the **Get started** page in New Relic.

Reviewer: Same comment as above, consider adding context as to where I need to follow these steps. Example:

"After installing our Android agent, you'll need to make some configurations to your Android app to successfully report data to New Relic. Follow the steps below in your app's Android Studio."

3. Set app permissions: Ensure that your Android app requests `INTERNET` and `ACCESS_NETWORK_STATE` permissions by adding these lines to your `AndroidManifest.xml` file:

  ```
  <uses-permission android:name="android.permission.INTERNET" />
  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
  ```
4. To start New Relic's mobile monitoring agent: In your **Default Activity** (as defined in your **Manifest**), import the `NewRelic` class:

  ```
  import com.newrelic.agent.android.NewRelic;
  ```

  <Callout variant="important">
    We do not support starting the mobile monitoring agent in other classes, as that can cause unexpected or unstable behavior.
  </Callout>
5. To initialize mobile monitoring: In the `onCreate()` method, add this call:

>   ```
>   NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplication());
>   ```

Reviewer: Let's add Java syntax highlighting to all these code samples (on the other install/configure Android pages too).
```suggestion
  ```java
  NewRelic.withApplicationToken("GENERATED_TOKEN").start(this.getApplication());
  ```
```
